logo/logo.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns="http://www.w3.org/2000/svg"
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
suggestion: remove all these `xmlns:xlink="http://www.w3.org/1999/xlink"` additions for minimalizing PR changes

Reply:
resolved in 16b32f85c53eaa8e7c29312e32d1299401eb69f9